### PR TITLE
Support aave usd tokens

### PR DESCRIPTION
**Pull request by xitronix**

### CODE REVIEW

File: universal-login-commons/src/core/services/TokenPricesService.ts [hunk 1]
```diff
@@ -2,16 +2,49 @@ import {ObservedCurrency, TokensPrices, TokenDetails, ETHER_NATIVE_TOKEN} from '
...
+
 export class TokenPricesService {
   constructor(private coingeckoApi = new CoingeckoApi()) {};
 
   async getPrices(tokensDetails: TokenDetails[]): Promise<TokensPrices> {
     const coingeckoTokensList = await this.coingeckoApi.lazyGetTokensList();
-    const tokenDetailsWithCoingeckoId = tokensDetails.map(token => ({...token, coingeckoId: this.coingeckoApi.findIdBySymbol(coingeckoTokensList, token)}));
+    const tokenDetailsWithCoingeckoId = tokensDetails
+      .map(this.updateAaveTokenDetails)
+      .map(token => ({...token, coingeckoId: this.coingeckoApi.findIdBySymbol(coingeckoTokensList, token)}))
```
Comment (JustynaBroniszewska) [inline universal-login-commons/src/core/services/TokenPricesService.ts:19]: We should extract function `addCoingecoId`

File: universal-login-commons/src/integration/http/CoingeckoApi.ts [hunk 2]
```diff
@@ -45,7 +45,9 @@ export class CoingeckoApi {
   };
 
   fetchTokenInfo = async (tokenDetails: TokenDetailsWithCoingeckoId[], currencies: ObservedCurrency[]) => {
-    const tokens = tokenDetails.map(token => token.coingeckoId);
+    const tokens = tokenDetails
+      .filter((token, index, tokens) => tokens.findIndex(t => t.symbol === token.symbol) === index)
```
Comment (JustynaBroniszewska) [inline universal-login-commons/src/integration/http/CoingeckoApi.ts:49]: why?

File: universal-login-commons/src/core/services/TokenPricesService.ts [hunk 3]
```diff
@@ -2,16 +2,49 @@ import {ObservedCurrency, TokensPrices, TokenDetails, ETHER_NATIVE_TOKEN} from '
...
+      .map(this.getOriginalAaveToken);
     const pricesWithCoingeckoId = await this.coingeckoApi.fetchTokenInfo(tokenDetailsWithCoingeckoId, ['ETH', 'USD']);
     return this.getPricesFromPricesWithCoingeckoId(tokenDetailsWithCoingeckoId, pricesWithCoingeckoId);
   }
 
+  updateAaveTokenDetails(token: TokenDetails) {
```
Comment (JustynaBroniszewska) [inline universal-login-commons/src/core/services/TokenPricesService.ts:25]: Could you please add any tests that will show it works?

File: universal-login-commons/src/integration/http/CoingeckoApi.ts (hunk 2 above)
Comment (xitronix) [reply universal-login-commons/src/integration/http/CoingeckoApi.ts:49]: We can't have duplicated tokens  when we call api

Comment (JustynaBroniszewska) [reply universal-login-commons/src/integration/http/CoingeckoApi.ts:49]: So then could we please extract function `removeDuplications`?

Comment (xitronix) [reply universal-login-commons/src/integration/http/CoingeckoApi.ts:49]: We can :))

**Pull request merged**